### PR TITLE
[FLINK-15434][Tests]Fix unstable tests in JobMasterTest

### PR DESCRIPTION
**Pull request by wangxlong**
## What is the purpose of the change

Fix unstable tests in JobMasterTest

## Brief change log

Create the jobmaster with heartbeatServices instead of fastHeartbeatServices for some tests.

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by wangxlong** [comment]
@flinkbot run travis

**Comment by wangxlong** [comment]
It is failed in python molule. I  run travis again.

**Comment by wangxlong** [comment]
@flinkbot run travis

**Comment by GJL** [comment]
Sorry, didn't have time to look at your PR yet but I will in the next days.

File: flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/JobMasterTest.java [hunk 1]
```diff
@@ -820,7 +822,8 @@ public void testCheckpointPrecedesSavepointRecovery() throws Exception {
 			configuration,
 			jobGraph,
 			haServices,
-			new TestingJobManagerSharedServicesBuilder().build());
+			new TestingJobManagerSharedServicesBuilder().build(),
+			heartbeatServices);
```
Comment (GJL) [inline flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/JobMasterTest.java]: Same here (only invoking the constructor)

**Comment by wangxlong** [comment]
Thank you @GJL. You are right, the other three tests only invoke the constructor, and no need to modify.

**Comment by GJL** [comment]
Merging.

**Pull request closed without merging**